domain/domain.go

```diff
@@ -170,7 +170,10 @@ func (do *Domain) fetchAllSchemasWithTables(m *meta.Meta) ([]*model.DBInfo, erro
 	return allSchemas, nil
 }
 
-const fetchSchemaConcurrency = 8
+// fetchSchemaConcurrency controls the goroutines to load schemas, but more goroutines
+// increase the memory usage when calling json.Unmarshal(), which would cause OOM,
+// so we decrease the concurrency.
+const fetchSchemaConcurrency = 1
```

Comment:
If set `fetchSchemaConcurrency` to 1, it seems that we needn't use goroutine to call `fetchSchemasWithTables` in `fetchAllSchemasWithTables`.
Is it too small to set `fetchSchemaConcurrency` to 1? And does it feel more appropriate to set it to 5?  Just don’t know the time consumption of the corresponding loading schema.
![WeChatWorkScreenshot_f3cb06fd-df97-4337-bf68-1596928237b4](https://user-images.githubusercontent.com/4242506/89608101-be791b00-d8a6-11ea-8ff7-72bbe99f9f50.png)

Reply:
I update the cost time in the table, it actually costs a little time, especially compared to other parts. As fetching schemas is call periodically, to reduce the risk of OOM carefully, we prefer choose the conncurrency to 1. In the worst case where there are many tables, like > 10000 tables, avoding OOM is more important to save loading time. In the general case where < 10000 tables, fetching schemas only cost several seconds, it does not domain the boost time.

Reply:
If set fetchSchemaConcurrency to 1, it seems that we needn't use goroutine to call fetchSchemasWithTables in fetchAllSchemasWithTables.
-------------
in case of increasing the concurrency.
We may set fetchSchemaConcurrency as a config parameter.

Reply:
OK